proposals/0000-rich-errors-proposal.rst

```diff
@@ -331,6 +407,27 @@ value represents a piece of the document which the consumer is free to
 render in any way it sees fit. All of the information relevant to the message
 is guaranteed to be in the embedded value.
 
+Moreover, it is easy to emulate scoped annotations with ``embed``-style
```

Comment:
Let's point out @mpickering's `kythe` work to motivation why we mention this:
```
However, some applications (e.g. TODO: link to kythe) are more natural to write with scoped-style annotations. For this cases it is possible to emulate scoped annotations with ``embed``-style document, ...
```

Reply:
Done.